.pre-commit-config.yaml

```diff
@@ -38,7 +38,7 @@ repos:
           - --profile
           - black
   - repo: https://github.com/psf/black
-    rev: 23.1.0
+    rev: 25.1.0
```

Comment:
I would have preferred that new formatting was in a separate PR.

Reply:
sure, can do. just annoying that precommit fails on my commits for unrelated changes 😅

Reply:
https://github.com/bazel-contrib/rules_python/pull/2808